src/database/seeders/20190702081652-demo_ratings.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert(
+      'Ratings',
```

Comment:
Good work so far @codexempire. Could you please avoid using seeders in the test database so we can have independent tests?